api/CMakeLists.txt

```diff
@@ -68,6 +68,7 @@ if (NOT PACK_INDIGO_NOT)
 endif()
 if (NOT DEFINED ENV{DISABLE_INDIGO_TESTS})
 DEFINE_TEST(indigo-c-test-shared "tests/c/indigo-test.c" indigo-shared)
+DEFINE_TEST(indigo-c-test-shared-issue-269 "tests/c/indigo-test-issue-269.c;tests/c/indigo-test-data-issue-269.cpp" indigo-shared)
```

Comment:
this tests are related to indigo c API. The tests for checker need to be moved to python tests

Reply:
OK but copied.
If something went wrong, these C tests would be a helpful entry point for C++ debugging.

Reply:
I would suggest to restore cpp unit tests. I will create additional PR for that

Reply:
Copied to Python. 
Done.